torchvision/models/optical_flow/raft.py

```diff
@@ -525,10 +525,12 @@ class Raft_Large_Weights(WeightsEnum):
             **_COMMON_META,
             "num_params": 5257536,
             "recipe": "https://github.com/princeton-vl/RAFT",
-            "sintel_train_cleanpass_epe": 1.4411,
-            "sintel_train_finalpass_epe": 2.7894,
-            "kitti_train_per_image_epe": 5.0172,
-            "kitti_train_f1-all": 17.4506,
+            "metrics": {
+                "sintel_train_cleanpass_epe": 1.4411,
```

Comment:
I'm not terribly happy with this approach but I left it as-is to get your input @NicolasHug. Happy to leave this on a follow up PR but here are some thoughts.

The need of providing accuracy metrics on multiple datasets is common and something we might face on other models (for example in Classification give metrics both for ImageNet1K V1/V2/Real datasets). In such cases, we might want to provide a map of accuracies one for each dataset. For example:
```python
"metrics": {
   # single metrics on the "best" available dataset (whichever this is for each weight) used for summarization
  "epe": 1.4411,
  "fl_all": 17.4506,
  # metrics across different datasets. Naming TBD
  "epe_breakdown": {
    "sintel_train_cleanpass": 1.4411,
    "sintel_train_finalpass": 2.7894,
    "kitti_train_per_image_epe": 5.0172,
  },
  "fl_all_breakdown": {
    "kitti_train": 17.4506,
   },
}
```

We need to figure out this detail before the release because making changes on the meta-data will be considered BC-breaking in our current definition.

Reply:
I agree, this is very adhoc for now. Happy to discuss this in the near future.